packages/node/src/indexer/benchmark.service.spec.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020-2022 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { INestApplication } from '@nestjs/common';
+import { EventEmitterModule } from '@nestjs/event-emitter';
+import { Test } from '@nestjs/testing';
+import { BlockHash } from '@polkadot/types/interfaces';
+import { GraphQLSchema } from 'graphql';
+import { SubqueryProject } from '../configure/SubqueryProject';
+import { delay } from '../utils/promise';
+import { ApiService } from './api.service';
+import { BenchmarkService } from './benchmark.service';
+
+jest.setTimeout(90000);
+describe('Benchmark service', () => {
+  let app: INestApplication;
+
+  afterEach(async () => {
+    return app?.close();
+  });
+
+  const newbenchmarkService = new BenchmarkService();
+
+  (newbenchmarkService as any).currentProcessingHeight = 1208163;
+  (newbenchmarkService as any).targetHeight = 1208163;
+  (newbenchmarkService as any).lastRegisteredHeight = 1208163;
+  (newbenchmarkService as any).lastRegisteredTimestamp = 10000;
+  (newbenchmarkService as any).currentProcessingTimestamp = 10000;
```

Comment:
This should be in a before each.

Also generally when we write tests they are setup so they are independent of one another and can run the single test.